CMakeLists.txt

```diff
@@ -491,7 +491,7 @@ FIND_PACKAGE(Samplerate 0.1.8 MODULE REQUIRED)
 
 # set compiler flags
 IF(CMAKE_CXX_COMPILER_ID MATCHES "GNU|Clang")
-	SET(WERROR_FLAGS "-Wall -Werror=unused-function -Wno-sign-compare -Wno-strict-overflow")
+	SET(WERROR_FLAGS "-Wall -Werror=unused-function -Wno-sign-compare -Wno-strict-overflow -Wdeprecated-register")
```

Comment:
Clang emits a ton of errors than this one. Also, did you mean `-Wno-deprecated-register`?

Reply:
Copying what I said on Discord:
> @Reflexe I'd rather add a `-DUSE_WERROR=OFF` to the macOS build.